seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java

```diff
@@ -17,7 +17,9 @@
 
 package org.apache.seatunnel.common.constants;
 
-public enum JobMode {
+import java.io.Serializable;
+
+public enum JobMode implements Serializable {
```

Comment:
revert


<img width="664" alt="image" src="https://github.com/apache/seatunnel/assets/14371345/ccc1f135-d489-4909-96ae-d2395b8ca1a9">

Reply:
![image](https://github.com/apache/seatunnel/assets/105560839/5452646f-b234-4bab-b7cc-ce8fcc78bc76)
In the above history which to revert

Reply:
revert this file uodates: `JobMode.java`